app/src/main/java/com/matmoongi/screens/LoginScreen.kt

```diff
@@ -50,12 +51,12 @@ fun LoginScreen(onClickSkipLoginButton: () -> Unit) {
             color = MaterialTheme.colorScheme.background,
             style = MaterialTheme.typography.bodyMedium,
         )
+
         Image(
             painter = painterResource(id = R.drawable.ic_naver_login),
             contentDescription = null,
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(top = 120.dp),
+            modifier = Modifier.fillMaxWidth()
+                .padding(top = 120.dp).clickable { onClickNaverLoginButton() },
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls

dot chain 줄바꿈 규칙 위반입니다.
```suggestion
            modifier = Modifier
                .fillMaxWidth()
                .padding(top = 120.dp)
                .clickable { onClickNaverLoginButton() },
```
or
```suggestion
            modifier = Modifier.fillMaxWidth()
                .padding(top = 120.dp)
                .clickable { onClickNaverLoginButton() },
```

Reply:
확실히 더 깔끔하네요